be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -8,7 +8,8 @@ Status AggregateBlockingSinkOperator::prepare(RuntimeState* state) {
     RETURN_IF_ERROR(Operator::prepare(state));
     // _aggregator is shared by sink operator and source operator
     // we must only prepare it at sink operator
-    return _aggregator->prepare(state, state->obj_pool(), get_memtracker(), get_runtime_profile());
+    return _aggregator->prepare(state, state->obj_pool(), get_memtracker(), get_memtracker(), nullptr,
```

Comment:
duplicate get_memtracker(), using one is just ok

Reply:
The prepare method is used in both vectorization and pipeline engines. In the case of vectorization engine, the two mem_tracker is different